src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -49,18 +52,19 @@ internal void CalculateEffectivePackageSaveMode()
                 packageSaveModeValue = SettingsUtility.GetConfigValue(Settings, "PackageSaveMode");
             }
 
-            EffectivePackageSaveMode = PackageSaveModes.None;
             if (!string.IsNullOrEmpty(packageSaveModeValue))
             {
+                // Restore \ Install always extract files
+                EffectivePackageSaveMode = Packaging.PackageSaveMode.Files;
```

Comment:
I'd wanted to let user make conscious decision about mode override value rather than alter the value on behalf of him/her.